dev/com.ibm.ws.beanvalidation.v20_fat/bnd.bnd

```diff
@@ -0,0 +1,25 @@
+#*******************************************************************************
+# Copyright (c) 2017 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
+#*******************************************************************************
+-include= ~../cnf/resources/bnd/bundle.props
+bVersion=1.0
+
+javac.source: 1.8
+javac.target: 1.8
+
+src: \
+	fat/src,\
+	test-applications/bvalApp/src
+
+test.project: true
+
+-buildpath: \
+	com.ibm.websphere.javaee.servlet.4.0;version=latest,\
+	com.ibm.websphere.javaee.validation.1.1;version=latest
```

Comment:
Should this be com.ibm.websphere.javaee.validation.2.0?

Reply:
yes it should, fixing